File: src/main/java/cn/xihan/lib/easyhttp/annotation/HttpHeader.java
package cn.xihan.lib.easyhttp.annotation;

import java.lang.annotation.*;

/**
 *    author : Android 轮子哥
 *    github : <a href="https://github.com/getActivity/EasyHttp">...</a>
 *    time   : 2019/05/19
 *    desc   : 请求头注解（标记这个字段是一个请求头的参数）
 */
@Documented
@Retention(RetentionPolicy.RUNTIME)
@Target(ElementType.FIELD)
public @interface HttpHeader {}

File: src/main/java/cn/xihan/lib/easyhttp/annotation/HttpHeader.kt
package cn.xihan.lib.easyhttp.annotation

/**
 * author : Android 轮子哥
 * github : [https://github.com/getActivity/EasyHttp](https://github.com/getActivity/EasyHttp)
 * time   : 2019/05/19
 * desc   : 请求头注解（标记这个字段是一个请求头的参数）
 */
@MustBeDocumented
@Retention(AnnotationRetention.RUNTIME)
@Target(AnnotationTarget.FIELD)
annotation class HttpHeader 